base/applications/cmdutils/attrib/attrib.c

```diff
@@ -371,20 +397,22 @@ int wmain(int argc, WCHAR *argv[])
     /* get full file name */
     for (i = 1; i < argc; i++)
     {
-        if ((*argv[i] != L'+') && (*argv[i] != L'-') && (*argv[i] != L'/'))
-        {
-            LPWSTR p;
+        if (*argv[i] == L'+' || *argv[i] == L'-' || *argv[i] == L'/')
+            continue;
 
-            GetFullPathName(argv[i], MAX_PATH, szPath, NULL);
-            p = wcsrchr(szPath, L'\\') + 1;
-            wcscpy(szFileName, p);
-            *p = L'\0';
+        GetFullPathNameW(argv[i], MAX_PATH, szPath, &p);
+        wcscpy(szFileName, p);
+        *p = 0;
 
-            if (dwMask == 0)
-                PrintAttribute(szPath, szFileName, bRecurse);
-            else
-                ChangeAttribute(szPath, szFileName, dwMask,
-                         dwAttrib, bRecurse, bDirectories);
+        if (dwMask == 0)
+        {
+            PrintAttribute(szPath, szFileName, bRecurse);
+        }
+        else if (!ChangeAttribute(szPath, szFileName, dwMask,
+                                  dwAttrib, bRecurse, bDirectories))
+        {
+            LoadStringW(NULL, STRING_FILE_NOT_FOUND, szText, ARRAYSIZE(szText));
+            ConPrintf(StdOut, szText, argv[i]);
```

Comment:
`ConResPrintf(StdErr, STRING_FILE_NOT_FOUND, argv[i]);`

Reply:
Fixed to `ConResPrintf(StdOut, STRING_FILE_NOT_FOUND, argv[i]);`.